Formula/gx-go.rb

```diff
@@ -0,0 +1,95 @@
+require "language/go"
+
+class GxGo < Formula
+  desc "Tool to use with the gx package manager for packages written in go."
+  homepage "https://github.com/whyrusleeping/gx-go"
+  url "https://github.com/whyrusleeping/gx-go.git",
+    :tag => "v1.0.0",
+    :revision => "e5258e8126435420207a15d914c895e076177af4"
+  head "https://github.com/whyrusleeping/gx-go.git"
+
+  depends_on "go" => :build
```

Comment:
I don't think it does (through the magic of go static binaries), but for safeties sake, for now we can depend on it until i get clarification or better understanding.